src/bot/cogs/bump.py

```
@@ -0,0 +1,133 @@
+        if not status or not status.get('enabled'):
+            return
+
+        is_success = False
+        if message.embeds:
+            desc = message.embeds[0].description or ""
+            if "表示順をアップ" in desc or "Bump done" in desc:
+                is_success = True
+
+        if is_success:
+            bot_id_str = str(message.author.id)
+            if not bot_id_str in self.ON_MESSAGE_BOTS:
```

Ruff (0.15.7)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)